src/testIntegration/kotlin/uk/gov/justice/digital/hmpps/sendlegalmailtoprisonsapi/magiclink/MagicLinkResourceTest.kt

```diff
@@ -46,7 +46,7 @@ class MagicLinkResourceTest(
         .uri("/link/email")
         .accept(MediaType.APPLICATION_JSON)
         .contentType(MediaType.APPLICATION_JSON)
-        .headers(setAuthorisation())
+        .headers(setAuthorisation(user = null))
```

Comment:
Changed all the create email / verify link calls to NOT pass in user_name because this is what happens in real life.